src/1a_prep_MPA.jl

```diff
@@ -15,35 +15,32 @@ compression.
 """
 
 include("common.jl")
+include("geom_handlers/raster_processing.jl")
+include("geom_handlers/lookup_processing.jl")
 
 # This processing step requires all the memory available so we actually remove the workers
 # that were just set up.
 # This can be commented out if working on a system with large enough amounts of memory
 rmprocs(workers()[2:end]...)
 GC.gc()
 
-using SparseArrays
+using SparseArrays, NamedTupleTools
 
 # Loading regions_4326 for cropping of vector and raster data.
 regions_4326 = GDF.read(REGION_PATH_4326)
 
 # 1. Process GBRMPA zoning geopackage to select only zones for site exclusion
-if !isfile(joinpath(MPA_OUTPUT_DIR, "GBRMPA_zone_exclusion.gpkg"))
-    GBRMPA_zoning_poly = GDF.read(
-        joinpath(
-            GDA2020_DATA_DIR,
-            "Great_Barrier_Reef_Marine_Park_Zoning_20_4418126048110066699.gpkg"
-        )
-    )
-    GBRMPA_zoning_poly = GBRMPA_zoning_poly[GBRMPA_zoning_poly.TYPE .∈ [MPA_EXCLUSION_ZONES], :]
-    rename!(GBRMPA_zoning_poly, :SHAPE => :geometry)
-
-    GDF.write(
-        joinpath(MPA_OUTPUT_DIR, "GBRMPA_zone_exclusion.gpkg"),
-        GBRMPA_zoning_poly;
-        crs=EPSG_7844
-    )
-end
+MPA_zoning_input = "$(GDA2020_DATA_DIR)/Great_Barrier_Reef_Marine_Park_Zoning_20_4418126048110066699.gpkg"
+MPA_preserv_zone_fn = joinpath(MPA_OUTPUT_DIR, "GBRMPA_preserv_zone_exclusion.gpkg")
```

Comment:
Noting the same here - might as well write out the full word:

```suggestion
MPA_preserv_zone_fn = joinpath(MPA_OUTPUT_DIR, "GBRMPA_preservation_zone_exclusion.gpkg")
```